github-profiles/Marcos.-0222/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Profile page</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</html>
+<head>
+<h1>🖐🏻Hello, I'm Marcos Viejo 🖐🏻
+    </h1>
+    </head>
+<body>
+    <h2>This is a briefly description about me</h2>
+    <h2>I'm a <strong>lawyer</strong> and working as <strong>Productivy Manager</strong> at the present
```

Comment:
Maybe this could be a `h3` tag instead another `h2` but this is up to you and page's design.